docs/workflows/user-provisioning.md

```diff
@@ -0,0 +1,155 @@
+---
+title: User Provisioning
+sidebar_position: 7
+
+# Display h2 to h2 headings
+toc_min_heading_level: 2
+toc_max_heading_level: 2
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# User Provisioning Overview
+
+Before creating a Passkey, a user must be provisioned as an identity under a realm. An identity is a unique identifier that may be used by an end-user to gain access governed by Beyond Identity. An end-user may have multiple identities. A realm can have many identities.
+
+Identities can be part of a group. A realm can have many groups. An identity can belong to many groups. Groups are commonly used as a predicate in a policy rule. (e.g. Allow group "Beyond Identity Administrators" to access the "Beyond Identity Admin Console" application)
+
+You can provision a user with:
+
+1. [Beyond Identity Admin Console](user-provisioning#provision-user-with-the-beyond-identity-admin-console)
+2. [Beyond Identity APIs](user-provisioning#provision-user-with-the-beyond-identity-api)
+3. [SCIM](../scim/v1/scimv1)
+
+## Provision User with the Beyond Identity Admin Console
+
+From the Beyond Identity Admin Console you can add identities and manage groups.
+
+### Identities
+
+The simplest way to create an identity is through the Beyond Identity Admin Console. First select a realm. Under the "PROJECT MANAGEMENT" tab, select "Identities", and then click on "Add identity". From there you can enter the user's name, username, and email.
+
+![Identities](./screenshots/user-provisioning-identities.png)
+
+![Add Identity](./screenshots/user-provisioning-add-identity.png)
+
+### Groups
+
+You can manage groups in the Beyond Identity Admin Console, creating groups, adding to and removing identities from groups. Under the "PROJECT MANAGEMENT" tab, select "Groups".
+
+![Groups](./screenshots/user-provisioning-groups.png)
+
+## Provision User with the Beyond Identity API
+
+Before making any API calls you'll want to generate an `accessToken`. Check out [Create an API token](./api-token) for help creating an access token.
+
+The following examples show how to create an identity via API. For deleting, updating, and managing groups via API, see the [API documentation](https://developer.beyondidentity.com/api/v1).
```

Comment:
```suggestion
The following examples show how to create an identity via the API. For deleting, updating, and managing groups via API, see the [API documentation](https://developer.beyondidentity.com/api/v1).
```

The last sentence is also a bit confusing since the section is about provisioning a user. Maybe it's enough to just link to the API docs (e.g. "for more information about the API, see <..>")?